CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 This document outlines major changes between releases.
 
+## 0.18.0 (16 Dec 2021)
+
+### Added
+- Support for MultipartUpload (#186, #187) 
+- CORS support (#217)
+- Authmate supports setting of tokens' lifetime in more convenient format (duration) (#258)
+- Generation of a random key for `--no-sign-request` (#276)
+
+### Changed
+- Bucket name resolving mechanism from listing owners' container to using DNS (#219)
```

Comment:
Maybe something like this would be better:
```suggestion
- Bucket name resolving mechanism from listing owner containers to using DNS (#219)
```

Reply:
Typo, but I think it'd be better `owner's containers`